rclip/main.py

```diff
@@ -228,6 +231,13 @@ def main():
         print(f'{r.score:.3f}\t"{r.filepath}"')
         if args.preview:
           preview(r.filepath, args.preview_height)
+        if os.path.exists(XNVIEW_PATH):
```

Comment:
Why does this always try to open the xnview?

1. What if I am on Linux/macOS?
2. What if I don't want to open xnview?